cmd/podman/images/scan.go

```diff
@@ -0,0 +1,117 @@
+package images
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/containers/podman/v2/cmd/podman/common"
+	"github.com/containers/podman/v2/cmd/podman/registry"
+	"github.com/containers/podman/v2/libpod/define"
+	"github.com/containers/podman/v2/pkg/domain/entities"
+	"github.com/containers/podman/v2/pkg/specgen"
+	"github.com/spf13/cobra"
+)
+
+var (
+	imageScanCommand     = &cobra.Command{
+		Use:     "scan [run-options] IMAGE [scanner-options]",
+		Short:   "Scan a container image for known vulnerabilities",
+		Long:    `Scan a container image for known vulnerabilities. The image name or digest can be used.`,
```

Comment:
```suggestion
		Long:    `Scan a container image for known vulnerabilities.`,
```
Image IDs should work as well.  I'd rather explain some of the details.  For instance, that the scanner itself is a container image and that the specified image to scan will be mounted inside.

Reply:
Still open.